L 10/13/2019 - 01:28:46: SourceMod error session started
L 10/13/2019 - 01:28:46: Info (map "c4m1_milltown_a") (file "errors_20191013.log")
L 10/13/2019 - 01:28:46: Error log file session closed.
L 10/13/2019 - 01:32:19: SourceMod error session started
L 10/13/2019 - 01:32:19: Info (map "c4m1_milltown_a") (file "errors_20191013.log")
L 10/13/2019 - 01:32:19: [SM] Exception reported: Client index 239 is invalid
L 10/13/2019 - 01:32:19: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 01:32:19: [SM] Call stack trace:
L 10/13/2019 - 01:32:19: [SM]   [0] IsClientInGame
L 10/13/2019 - 01:32:19: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 01:32:19: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 01:42:28: Error log file session closed.
L 10/13/2019 - 01:46:43: SourceMod error session started
L 10/13/2019 - 01:46:43: Info (map "c4m2_sugarmill_a") (file "errors_20191013.log")
L 10/13/2019 - 01:46:43: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 01:46:43: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 01:46:43: [SM] Call stack trace:
L 10/13/2019 - 01:46:43: [SM]   [0] SetEntProp
L 10/13/2019 - 01:46:43: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 01:55:03: Error log file session closed.
L 10/13/2019 - 02:05:01: SourceMod error session started
L 10/13/2019 - 02:05:01: Info (map "c4m3_sugarmill_b") (file "errors_20191013.log")
L 10/13/2019 - 02:05:01: [SM] Exception reported: Entity 9 (9) is invalid
L 10/13/2019 - 02:05:01: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 02:05:01: [SM] Call stack trace:
L 10/13/2019 - 02:05:01: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 02:05:01: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 02:08:46: [SM] Exception reported: Client index 95 is invalid
L 10/13/2019 - 02:08:46: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 02:08:46: [SM] Call stack trace:
L 10/13/2019 - 02:08:46: [SM]   [0] IsClientInGame
L 10/13/2019 - 02:08:46: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 02:08:46: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 02:13:32: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 02:13:32: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 02:13:32: [SM] Call stack trace:
L 10/13/2019 - 02:13:32: [SM]   [0] SetEntProp
L 10/13/2019 - 02:13:32: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 02:17:33: Error log file session closed.
L 10/13/2019 - 02:26:43: SourceMod error session started
L 10/13/2019 - 02:26:43: Info (map "c2m1_highway") (file "errors_20191013.log")
L 10/13/2019 - 02:26:43: [CRASH] Accelerator uploaded crash dump: Crash ID: Q2NX-APSB-BCSE
L 10/13/2019 - 02:57:21: Error log file session closed.
L 10/13/2019 - 03:02:16: SourceMod error session started
L 10/13/2019 - 03:02:16: Info (map "c2m1_highway") (file "errors_20191013.log")
L 10/13/2019 - 03:02:16: [SM] Exception reported: Client index 353 is invalid
L 10/13/2019 - 03:02:16: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 03:02:16: [SM] Call stack trace:
L 10/13/2019 - 03:02:16: [SM]   [0] IsClientInGame
L 10/13/2019 - 03:02:16: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 03:02:16: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 03:14:24: [SM] Exception reported: Client 3 is not in game
L 10/13/2019 - 03:14:24: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 03:14:24: [SM] Call stack trace:
L 10/13/2019 - 03:14:24: [SM]   [0] GetClientTeam
L 10/13/2019 - 03:14:24: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 03:15:54: [SM] Exception reported: Entity 1 (1) is invalid
L 10/13/2019 - 03:15:54: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 03:15:54: [SM] Call stack trace:
L 10/13/2019 - 03:15:54: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 03:15:54: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 03:15:54: Error log file session closed.
L 10/13/2019 - 03:20:56: SourceMod error session started
L 10/13/2019 - 03:20:56: Info (map "c2m2_fairgrounds") (file "errors_20191013.log")
L 10/13/2019 - 03:20:56: [SM] Exception reported: Client 2 is not in game
L 10/13/2019 - 03:20:56: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 03:20:56: [SM] Call stack trace:
L 10/13/2019 - 03:20:56: [SM]   [0] GetClientTeam
L 10/13/2019 - 03:20:56: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 03:28:45: Error log file session closed.
L 10/13/2019 - 03:43:11: SourceMod error session started
L 10/13/2019 - 03:43:11: Info (map "c2m4_barns") (file "errors_20191013.log")
L 10/13/2019 - 03:43:11: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 03:43:11: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 03:43:11: [SM] Call stack trace:
L 10/13/2019 - 03:43:11: [SM]   [0] SetEntProp
L 10/13/2019 - 03:43:11: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 03:44:20: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 03:44:20: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 03:44:20: [SM] Call stack trace:
L 10/13/2019 - 03:44:20: [SM]   [0] SetEntProp
L 10/13/2019 - 03:44:20: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 03:44:57: [SM] Exception reported: Entity 11 (11) is invalid
L 10/13/2019 - 03:44:57: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 03:44:57: [SM] Call stack trace:
L 10/13/2019 - 03:44:57: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 03:44:57: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 03:44:57: Error log file session closed.
L 10/13/2019 - 03:46:31: SourceMod error session started
L 10/13/2019 - 03:46:31: Info (map "c2m5_concert") (file "errors_20191013.log")
L 10/13/2019 - 03:46:31: [SM] Exception reported: Client index 527 is invalid
L 10/13/2019 - 03:46:31: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 03:46:31: [SM] Call stack trace:
L 10/13/2019 - 03:46:31: [SM]   [0] IsClientInGame
L 10/13/2019 - 03:46:31: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 03:46:31: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 03:46:31: [SM] Exception reported: Client index 527 is invalid
L 10/13/2019 - 03:46:31: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 03:46:31: [SM] Call stack trace:
L 10/13/2019 - 03:46:31: [SM]   [0] IsClientInGame
L 10/13/2019 - 03:46:31: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 03:46:31: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 03:46:48: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 03:46:48: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 03:46:48: [SM] Call stack trace:
L 10/13/2019 - 03:46:48: [SM]   [0] SetEntProp
L 10/13/2019 - 03:46:48: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 03:49:05: [SM] Exception reported: Client index 288 is invalid
L 10/13/2019 - 03:49:05: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 03:49:05: [SM] Call stack trace:
L 10/13/2019 - 03:49:05: [SM]   [0] IsClientInGame
L 10/13/2019 - 03:49:05: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 03:49:05: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 03:50:58: [SM] Exception reported: Entity 1 (1) is invalid
L 10/13/2019 - 03:50:58: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 03:50:58: [SM] Call stack trace:
L 10/13/2019 - 03:50:58: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 03:50:58: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 03:50:58: [SM] Exception reported: Entity 8 (8) is invalid
L 10/13/2019 - 03:50:58: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 03:50:58: [SM] Call stack trace:
L 10/13/2019 - 03:50:58: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 03:50:58: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 03:50:58: Error log file session closed.
L 10/13/2019 - 03:51:01: SourceMod error session started
L 10/13/2019 - 03:51:01: Info (map "c5m1_waterfront") (file "errors_20191013.log")
L 10/13/2019 - 03:51:01: [SM] Exception reported: Client 5 is not in game
L 10/13/2019 - 03:51:01: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 03:51:01: [SM] Call stack trace:
L 10/13/2019 - 03:51:01: [SM]   [0] GetClientTeam
L 10/13/2019 - 03:51:01: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 03:57:55: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 03:57:55: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 03:57:55: [SM] Call stack trace:
L 10/13/2019 - 03:57:55: [SM]   [0] SetEntProp
L 10/13/2019 - 03:57:55: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 03:58:20: Error log file session closed.
L 10/13/2019 - 04:01:31: SourceMod error session started
L 10/13/2019 - 04:01:31: Info (map "c5m2_park") (file "errors_20191013.log")
L 10/13/2019 - 04:01:31: [SM] Exception reported: Client index 364 is invalid
L 10/13/2019 - 04:01:31: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 04:01:31: [SM] Call stack trace:
L 10/13/2019 - 04:01:31: [SM]   [0] IsClientInGame
L 10/13/2019 - 04:01:31: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 04:01:31: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 04:06:37: [SM] Exception reported: Client index 140 is invalid
L 10/13/2019 - 04:06:37: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 04:06:37: [SM] Call stack trace:
L 10/13/2019 - 04:06:37: [SM]   [0] IsClientInGame
L 10/13/2019 - 04:06:37: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 04:06:37: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 04:10:52: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 04:10:52: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 04:10:52: [SM] Call stack trace:
L 10/13/2019 - 04:10:52: [SM]   [0] SetEntProp
L 10/13/2019 - 04:10:52: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 04:11:46: [SM] Exception reported: Entity 1 (1) is invalid
L 10/13/2019 - 04:11:46: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 04:11:46: [SM] Call stack trace:
L 10/13/2019 - 04:11:46: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 04:11:46: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 04:11:48: Error log file session closed.
L 10/13/2019 - 04:24:40: SourceMod error session started
L 10/13/2019 - 04:24:40: Info (map "c5m3_cemetery") (file "errors_20191013.log")
L 10/13/2019 - 04:24:40: [SM] Exception reported: Entity 5 (5) is invalid
L 10/13/2019 - 04:24:40: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 04:24:40: [SM] Call stack trace:
L 10/13/2019 - 04:24:40: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 04:24:40: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 04:29:30: Error log file session closed.
L 10/13/2019 - 04:37:22: SourceMod error session started
L 10/13/2019 - 04:37:22: Info (map "c5m4_quarter") (file "errors_20191013.log")
L 10/13/2019 - 04:37:22: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 04:37:22: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 04:37:22: [SM] Call stack trace:
L 10/13/2019 - 04:37:22: [SM]   [0] SetEntProp
L 10/13/2019 - 04:37:22: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 04:47:02: Error log file session closed.
L 10/13/2019 - 04:57:27: SourceMod error session started
L 10/13/2019 - 04:57:27: Info (map "c5m5_bridge") (file "errors_20191013.log")
L 10/13/2019 - 04:57:27: [SM] Exception reported: Entity 3 (3) is invalid
L 10/13/2019 - 04:57:27: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 04:57:27: [SM] Call stack trace:
L 10/13/2019 - 04:57:27: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 04:57:27: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 04:57:27: Error log file session closed.
L 10/13/2019 - 05:03:58: SourceMod error session started
L 10/13/2019 - 05:03:58: Info (map "c1m1_hotel") (file "errors_20191013.log")
L 10/13/2019 - 05:03:58: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 05:03:58: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 05:03:58: [SM] Call stack trace:
L 10/13/2019 - 05:03:58: [SM]   [0] SetEntProp
L 10/13/2019 - 05:03:58: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 05:04:32: Error log file session closed.
L 10/13/2019 - 05:12:24: SourceMod error session started
L 10/13/2019 - 05:12:24: Info (map "c1m2_streets") (file "errors_20191013.log")
L 10/13/2019 - 05:12:24: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 05:12:24: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 05:12:24: [SM] Call stack trace:
L 10/13/2019 - 05:12:24: [SM]   [0] SetEntProp
L 10/13/2019 - 05:12:24: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 05:20:01: Error log file session closed.
L 10/13/2019 - 05:45:02: SourceMod error session started
L 10/13/2019 - 05:45:02: Info (map "c1m4_atrium") (file "errors_20191013.log")
L 10/13/2019 - 05:45:02: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 05:45:02: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 05:45:02: [SM] Call stack trace:
L 10/13/2019 - 05:45:02: [SM]   [0] SetEntProp
L 10/13/2019 - 05:45:02: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 05:47:42: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 05:47:42: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 05:47:42: [SM] Call stack trace:
L 10/13/2019 - 05:47:42: [SM]   [0] SetEntProp
L 10/13/2019 - 05:47:42: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 05:48:32: [SM] Exception reported: Entity 3 (3) is invalid
L 10/13/2019 - 05:48:32: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 05:48:32: [SM] Call stack trace:
L 10/13/2019 - 05:48:32: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 05:48:32: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 05:48:32: [SM] Exception reported: Entity 7 (7) is invalid
L 10/13/2019 - 05:48:32: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 05:48:32: [SM] Call stack trace:
L 10/13/2019 - 05:48:32: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 05:48:32: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 05:48:33: Error log file session closed.
L 10/13/2019 - 05:48:40: SourceMod error session started
L 10/13/2019 - 05:48:40: Info (map "c2m1_highway") (file "errors_20191013.log")
L 10/13/2019 - 05:48:40: [SM] Exception reported: Client 10 is not in game
L 10/13/2019 - 05:48:40: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 05:48:40: [SM] Call stack trace:
L 10/13/2019 - 05:48:40: [SM]   [0] GetClientTeam
L 10/13/2019 - 05:48:40: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 05:49:52: [SM] Exception reported: Client 3 is not in game
L 10/13/2019 - 05:49:52: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 05:49:52: [SM] Call stack trace:
L 10/13/2019 - 05:49:52: [SM]   [0] GetClientTeam
L 10/13/2019 - 05:49:52: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 05:51:19: SourceMod error session started
L 10/13/2019 - 05:51:19: Info (map "c2m1_highway") (file "errors_20191013.log")
L 10/13/2019 - 05:51:19: [CRASH] Accelerator uploaded crash dump: Crash ID: LPRO-ZKEQ-K2SO
L 10/13/2019 - 10:25:10: Error log file session closed.
L 10/13/2019 - 13:07:43: SourceMod error session started
L 10/13/2019 - 13:07:43: Info (map "c2m1_highway") (file "errors_20191013.log")
L 10/13/2019 - 13:07:43: [SM] Exception reported: Client index 291 is invalid
L 10/13/2019 - 13:07:43: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 13:07:43: [SM] Call stack trace:
L 10/13/2019 - 13:07:43: [SM]   [0] IsClientInGame
L 10/13/2019 - 13:07:43: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 13:07:43: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 13:13:35: Error log file session closed.
L 10/13/2019 - 13:27:49: SourceMod error session started
L 10/13/2019 - 13:27:49: Info (map "c2m2_fairgrounds") (file "errors_20191013.log")
L 10/13/2019 - 13:27:49: [SM] Exception reported: Entity 10 (10) is invalid
L 10/13/2019 - 13:27:49: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 13:27:49: [SM] Call stack trace:
L 10/13/2019 - 13:27:49: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 13:27:49: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 13:27:50: Error log file session closed.
L 10/13/2019 - 13:34:45: SourceMod error session started
L 10/13/2019 - 13:34:45: Info (map "c2m3_coaster") (file "errors_20191013.log")
L 10/13/2019 - 13:34:45: [SM] Exception reported: Client index 349 is invalid
L 10/13/2019 - 13:34:45: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 13:34:45: [SM] Call stack trace:
L 10/13/2019 - 13:34:45: [SM]   [0] IsClientInGame
L 10/13/2019 - 13:34:45: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 13:34:45: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 13:34:46: [SM] Exception reported: Client index 349 is invalid
L 10/13/2019 - 13:34:46: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 13:34:46: [SM] Call stack trace:
L 10/13/2019 - 13:34:46: [SM]   [0] IsClientInGame
L 10/13/2019 - 13:34:46: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 13:34:46: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 13:34:50: [SM] Exception reported: Client index 349 is invalid
L 10/13/2019 - 13:34:50: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 13:34:50: [SM] Call stack trace:
L 10/13/2019 - 13:34:50: [SM]   [0] IsClientInGame
L 10/13/2019 - 13:34:50: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 13:34:50: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 13:36:32: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 13:36:32: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 13:36:32: [SM] Call stack trace:
L 10/13/2019 - 13:36:32: [SM]   [0] SetEntProp
L 10/13/2019 - 13:36:32: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 13:36:33: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 13:36:33: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 13:36:33: [SM] Call stack trace:
L 10/13/2019 - 13:36:33: [SM]   [0] SetEntProp
L 10/13/2019 - 13:36:33: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 13:37:10: Error log file session closed.
L 10/13/2019 - 13:39:32: SourceMod error session started
L 10/13/2019 - 13:39:32: Info (map "c2m4_barns") (file "errors_20191013.log")
L 10/13/2019 - 13:39:32: [SM] Exception reported: Client index 215 is invalid
L 10/13/2019 - 13:39:32: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 13:39:32: [SM] Call stack trace:
L 10/13/2019 - 13:39:32: [SM]   [0] IsClientInGame
L 10/13/2019 - 13:39:32: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 13:39:32: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 13:46:13: [SM] Exception reported: Entity 6 (6) is invalid
L 10/13/2019 - 13:46:13: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 13:46:13: [SM] Call stack trace:
L 10/13/2019 - 13:46:13: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 13:46:13: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 13:52:00: Error log file session closed.
L 10/13/2019 - 13:55:30: SourceMod error session started
L 10/13/2019 - 13:55:30: Info (map "c2m5_concert") (file "errors_20191013.log")
L 10/13/2019 - 13:55:30: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 13:55:30: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 13:55:30: [SM] Call stack trace:
L 10/13/2019 - 13:55:30: [SM]   [0] SetEntProp
L 10/13/2019 - 13:55:30: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 13:55:32: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 13:55:32: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 13:55:32: [SM] Call stack trace:
L 10/13/2019 - 13:55:32: [SM]   [0] SetEntProp
L 10/13/2019 - 13:55:32: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 13:58:09: [SM] Exception reported: Client index 500 is invalid
L 10/13/2019 - 13:58:09: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 13:58:09: [SM] Call stack trace:
L 10/13/2019 - 13:58:09: [SM]   [0] IsClientInGame
L 10/13/2019 - 13:58:09: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 13:58:09: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 13:58:11: [SM] Exception reported: Client index 500 is invalid
L 10/13/2019 - 13:58:11: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 13:58:11: [SM] Call stack trace:
L 10/13/2019 - 13:58:11: [SM]   [0] IsClientInGame
L 10/13/2019 - 13:58:11: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 13:58:11: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 13:58:11: [SM] Exception reported: Client index 500 is invalid
L 10/13/2019 - 13:58:11: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 13:58:11: [SM] Call stack trace:
L 10/13/2019 - 13:58:11: [SM]   [0] IsClientInGame
L 10/13/2019 - 13:58:11: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 13:58:11: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 13:58:13: [SM] Exception reported: Client index 500 is invalid
L 10/13/2019 - 13:58:13: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 13:58:13: [SM] Call stack trace:
L 10/13/2019 - 13:58:13: [SM]   [0] IsClientInGame
L 10/13/2019 - 13:58:13: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 13:58:13: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 13:59:37: [SM] Exception reported: Entity 2 (2) is invalid
L 10/13/2019 - 13:59:37: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 13:59:37: [SM] Call stack trace:
L 10/13/2019 - 13:59:37: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 13:59:37: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 13:59:37: [SM] Exception reported: Entity 3 (3) is invalid
L 10/13/2019 - 13:59:37: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 13:59:37: [SM] Call stack trace:
L 10/13/2019 - 13:59:37: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 13:59:37: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 13:59:38: Error log file session closed.
L 10/13/2019 - 14:02:36: SourceMod error session started
L 10/13/2019 - 14:02:36: Info (map "c5m1_waterfront") (file "errors_20191013.log")
L 10/13/2019 - 14:02:36: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:02:36: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:02:36: [SM] Call stack trace:
L 10/13/2019 - 14:02:36: [SM]   [0] SetEntProp
L 10/13/2019 - 14:02:36: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:04:17: [SM] Exception reported: Client index 452 is invalid
L 10/13/2019 - 14:04:17: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 14:04:17: [SM] Call stack trace:
L 10/13/2019 - 14:04:17: [SM]   [0] IsClientInGame
L 10/13/2019 - 14:04:17: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 14:04:17: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 14:08:53: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:08:53: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:08:53: [SM] Call stack trace:
L 10/13/2019 - 14:08:53: [SM]   [0] SetEntProp
L 10/13/2019 - 14:08:53: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:09:03: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:09:03: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:09:03: [SM] Call stack trace:
L 10/13/2019 - 14:09:03: [SM]   [0] SetEntProp
L 10/13/2019 - 14:09:03: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:09:31: [SM] Exception reported: Client 4 is not in game
L 10/13/2019 - 14:09:31: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 14:09:31: [SM] Call stack trace:
L 10/13/2019 - 14:09:31: [SM]   [0] GetClientTeam
L 10/13/2019 - 14:09:31: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 14:09:31: [SM] Exception reported: Client 11 is not in game
L 10/13/2019 - 14:09:31: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 14:09:31: [SM] Call stack trace:
L 10/13/2019 - 14:09:31: [SM]   [0] GetClientTeam
L 10/13/2019 - 14:09:31: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 14:09:32: Error log file session closed.
L 10/13/2019 - 14:12:12: SourceMod error session started
L 10/13/2019 - 14:12:12: Info (map "c5m2_park") (file "errors_20191013.log")
L 10/13/2019 - 14:12:12: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:12:12: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:12:12: [SM] Call stack trace:
L 10/13/2019 - 14:12:12: [SM]   [0] SetEntProp
L 10/13/2019 - 14:12:12: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:17:24: [SM] Exception reported: Client index 99 is invalid
L 10/13/2019 - 14:17:24: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 14:17:24: [SM] Call stack trace:
L 10/13/2019 - 14:17:24: [SM]   [0] IsClientInGame
L 10/13/2019 - 14:17:24: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 14:17:24: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 14:24:06: [SM] Exception reported: Client index 341 is invalid
L 10/13/2019 - 14:24:06: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 14:24:06: [SM] Call stack trace:
L 10/13/2019 - 14:24:06: [SM]   [0] IsClientInGame
L 10/13/2019 - 14:24:06: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 14:24:06: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 14:24:09: [SM] Exception reported: Client index 341 is invalid
L 10/13/2019 - 14:24:09: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 14:24:09: [SM] Call stack trace:
L 10/13/2019 - 14:24:09: [SM]   [0] IsClientInGame
L 10/13/2019 - 14:24:09: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 14:24:09: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 14:24:45: Error log file session closed.
L 10/13/2019 - 14:27:26: SourceMod error session started
L 10/13/2019 - 14:27:26: Info (map "c5m3_cemetery") (file "errors_20191013.log")
L 10/13/2019 - 14:27:26: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:27:26: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:27:26: [SM] Call stack trace:
L 10/13/2019 - 14:27:26: [SM]   [0] SetEntProp
L 10/13/2019 - 14:27:26: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:27:31: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:27:31: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:27:31: [SM] Call stack trace:
L 10/13/2019 - 14:27:31: [SM]   [0] SetEntProp
L 10/13/2019 - 14:27:31: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:31:49: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:31:49: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:31:49: [SM] Call stack trace:
L 10/13/2019 - 14:31:49: [SM]   [0] SetEntProp
L 10/13/2019 - 14:31:49: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:32:00: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:32:00: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:32:00: [SM] Call stack trace:
L 10/13/2019 - 14:32:00: [SM]   [0] SetEntProp
L 10/13/2019 - 14:32:00: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:32:21: Error log file session closed.
L 10/13/2019 - 14:42:46: SourceMod error session started
L 10/13/2019 - 14:42:46: Info (map "c5m4_quarter") (file "errors_20191013.log")
L 10/13/2019 - 14:42:46: [SM] Exception reported: Entity 11 (11) is invalid
L 10/13/2019 - 14:42:46: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 14:42:46: [SM] Call stack trace:
L 10/13/2019 - 14:42:46: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 14:42:46: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 14:47:03: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:47:03: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:47:03: [SM] Call stack trace:
L 10/13/2019 - 14:47:03: [SM]   [0] SetEntProp
L 10/13/2019 - 14:47:03: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:47:27: [SM] Exception reported: Client 12 is not in game
L 10/13/2019 - 14:47:27: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 14:47:27: [SM] Call stack trace:
L 10/13/2019 - 14:47:27: [SM]   [0] GetClientTeam
L 10/13/2019 - 14:47:27: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 14:47:27: Error log file session closed.
L 10/13/2019 - 14:49:28: SourceMod error session started
L 10/13/2019 - 14:49:28: Info (map "c5m5_bridge") (file "errors_20191013.log")
L 10/13/2019 - 14:49:28: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:49:28: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:49:28: [SM] Call stack trace:
L 10/13/2019 - 14:49:28: [SM]   [0] SetEntProp
L 10/13/2019 - 14:49:28: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:49:37: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:49:37: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:49:37: [SM] Call stack trace:
L 10/13/2019 - 14:49:37: [SM]   [0] SetEntProp
L 10/13/2019 - 14:49:37: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:52:26: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:52:26: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:52:26: [SM] Call stack trace:
L 10/13/2019 - 14:52:26: [SM]   [0] SetEntProp
L 10/13/2019 - 14:52:26: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:52:56: [SM] Exception reported: Entity 4 (4) is invalid
L 10/13/2019 - 14:52:56: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 14:52:56: [SM] Call stack trace:
L 10/13/2019 - 14:52:56: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 14:52:56: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 14:52:56: Error log file session closed.
L 10/13/2019 - 14:53:07: SourceMod error session started
L 10/13/2019 - 14:53:07: Info (map "c1m1_hotel") (file "errors_20191013.log")
L 10/13/2019 - 14:53:07: [SM] Exception reported: Client 8 is not in game
L 10/13/2019 - 14:53:07: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 14:53:07: [SM] Call stack trace:
L 10/13/2019 - 14:53:07: [SM]   [0] GetClientTeam
L 10/13/2019 - 14:53:07: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 14:55:08: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:55:08: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:55:08: [SM] Call stack trace:
L 10/13/2019 - 14:55:08: [SM]   [0] SetEntProp
L 10/13/2019 - 14:55:08: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:55:27: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 14:55:27: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 14:55:27: [SM] Call stack trace:
L 10/13/2019 - 14:55:27: [SM]   [0] SetEntProp
L 10/13/2019 - 14:55:27: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 14:58:23: [SM] Exception reported: Client index 358 is invalid
L 10/13/2019 - 14:58:23: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 14:58:23: [SM] Call stack trace:
L 10/13/2019 - 14:58:23: [SM]   [0] IsClientInGame
L 10/13/2019 - 14:58:23: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 14:58:23: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 14:58:23: [SM] Exception reported: Client index 358 is invalid
L 10/13/2019 - 14:58:23: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 14:58:23: [SM] Call stack trace:
L 10/13/2019 - 14:58:23: [SM]   [0] IsClientInGame
L 10/13/2019 - 14:58:23: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 14:58:23: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 15:01:54: [SM] Exception reported: Client 4 is not in game
L 10/13/2019 - 15:01:54: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 15:01:54: [SM] Call stack trace:
L 10/13/2019 - 15:01:54: [SM]   [0] GetClientTeam
L 10/13/2019 - 15:01:54: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 15:01:54: Error log file session closed.
L 10/13/2019 - 15:08:25: SourceMod error session started
L 10/13/2019 - 15:08:25: Info (map "c1m2_streets") (file "errors_20191013.log")
L 10/13/2019 - 15:08:25: [SM] Exception reported: Client 5 is not in game
L 10/13/2019 - 15:08:25: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 15:08:25: [SM] Call stack trace:
L 10/13/2019 - 15:08:25: [SM]   [0] GetClientTeam
L 10/13/2019 - 15:08:25: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 15:15:42: [SM] Exception reported: Client 8 is not in game
L 10/13/2019 - 15:15:42: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 15:15:42: [SM] Call stack trace:
L 10/13/2019 - 15:15:42: [SM]   [0] GetClientTeam
L 10/13/2019 - 15:15:42: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 15:15:42: Error log file session closed.
L 10/13/2019 - 15:18:03: SourceMod error session started
L 10/13/2019 - 15:18:03: Info (map "c1m3_mall") (file "errors_20191013.log")
L 10/13/2019 - 15:18:03: [SM] Exception reported: Client 4 is not in game
L 10/13/2019 - 15:18:03: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 15:18:03: [SM] Call stack trace:
L 10/13/2019 - 15:18:03: [SM]   [0] GetClientTeam
L 10/13/2019 - 15:18:03: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 15:22:56: [SM] Exception reported: Entity 3 (3) is invalid
L 10/13/2019 - 15:22:56: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 15:22:56: [SM] Call stack trace:
L 10/13/2019 - 15:22:56: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 15:22:56: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 15:29:32: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 15:29:32: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 15:29:32: [SM] Call stack trace:
L 10/13/2019 - 15:29:32: [SM]   [0] SetEntProp
L 10/13/2019 - 15:29:32: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 15:32:38: [SM] Exception reported: Entity 1 (1) is invalid
L 10/13/2019 - 15:32:38: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 15:32:38: [SM] Call stack trace:
L 10/13/2019 - 15:32:38: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 15:32:38: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 15:32:38: [SM] Exception reported: Entity 7 (7) is invalid
L 10/13/2019 - 15:32:38: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 15:32:38: [SM] Call stack trace:
L 10/13/2019 - 15:32:38: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 15:32:38: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 15:32:38: Error log file session closed.
L 10/13/2019 - 15:34:53: SourceMod error session started
L 10/13/2019 - 15:34:53: Info (map "c1m4_atrium") (file "errors_20191013.log")
L 10/13/2019 - 15:34:53: [SM] Exception reported: Client index 500 is invalid
L 10/13/2019 - 15:34:53: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 15:34:53: [SM] Call stack trace:
L 10/13/2019 - 15:34:53: [SM]   [0] IsClientInGame
L 10/13/2019 - 15:34:53: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 15:34:53: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 15:38:20: [SM] Exception reported: Client index 143 is invalid
L 10/13/2019 - 15:38:20: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 15:38:20: [SM] Call stack trace:
L 10/13/2019 - 15:38:20: [SM]   [0] IsClientInGame
L 10/13/2019 - 15:38:20: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 15:38:20: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 15:43:18: [SM] Exception reported: Entity 3 (3) is invalid
L 10/13/2019 - 15:43:18: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 15:43:18: [SM] Call stack trace:
L 10/13/2019 - 15:43:18: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 15:43:18: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 15:43:18: [SM] Exception reported: Client 7 is not in game
L 10/13/2019 - 15:43:18: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 15:43:18: [SM] Call stack trace:
L 10/13/2019 - 15:43:18: [SM]   [0] GetClientTeam
L 10/13/2019 - 15:43:18: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 15:43:18: [SM] Exception reported: Entity 6 (6) is invalid
L 10/13/2019 - 15:43:18: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 15:43:18: [SM] Call stack trace:
L 10/13/2019 - 15:43:18: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 15:43:18: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 15:43:18: Error log file session closed.
L 10/13/2019 - 15:45:01: SourceMod error session started
L 10/13/2019 - 15:45:01: Info (map "c1m1_hotel") (file "errors_20191013.log")
L 10/13/2019 - 15:45:01: [SM] Exception reported: Client index 50 is invalid
L 10/13/2019 - 15:45:01: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 15:45:01: [SM] Call stack trace:
L 10/13/2019 - 15:45:01: [SM]   [0] IsClientInGame
L 10/13/2019 - 15:45:01: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 15:45:01: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 15:45:03: [SM] Exception reported: Client index 50 is invalid
L 10/13/2019 - 15:45:03: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 15:45:03: [SM] Call stack trace:
L 10/13/2019 - 15:45:03: [SM]   [0] IsClientInGame
L 10/13/2019 - 15:45:03: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 15:45:03: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 15:48:00: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 15:48:00: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 15:48:00: [SM] Call stack trace:
L 10/13/2019 - 15:48:00: [SM]   [0] SetEntProp
L 10/13/2019 - 15:48:00: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 15:49:28: [SM] Exception reported: Client index 72 is invalid
L 10/13/2019 - 15:49:28: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 15:49:28: [SM] Call stack trace:
L 10/13/2019 - 15:49:28: [SM]   [0] IsClientInGame
L 10/13/2019 - 15:49:28: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 15:49:28: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 15:49:29: [SM] Exception reported: Client index 72 is invalid
L 10/13/2019 - 15:49:29: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 15:49:29: [SM] Call stack trace:
L 10/13/2019 - 15:49:29: [SM]   [0] IsClientInGame
L 10/13/2019 - 15:49:29: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 15:49:29: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 15:49:37: [SM] Exception reported: Client index 72 is invalid
L 10/13/2019 - 15:49:37: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 15:49:37: [SM] Call stack trace:
L 10/13/2019 - 15:49:37: [SM]   [0] IsClientInGame
L 10/13/2019 - 15:49:37: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 15:49:37: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 15:52:20: Error log file session closed.
L 10/13/2019 - 16:08:56: SourceMod error session started
L 10/13/2019 - 16:08:56: Info (map "c1m3_mall") (file "errors_20191013.log")
L 10/13/2019 - 16:08:56: [SM] Exception reported: Entity 9 (9) is invalid
L 10/13/2019 - 16:08:56: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 16:08:56: [SM] Call stack trace:
L 10/13/2019 - 16:08:56: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 16:08:56: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 16:08:56: [SM] Exception reported: Entity 8 (8) is invalid
L 10/13/2019 - 16:08:56: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 16:08:56: [SM] Call stack trace:
L 10/13/2019 - 16:08:56: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 16:08:56: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 16:11:42: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 16:11:42: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 16:11:42: [SM] Call stack trace:
L 10/13/2019 - 16:11:42: [SM]   [0] SetEntProp
L 10/13/2019 - 16:11:42: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 16:12:01: [SM] Exception reported: Client 3 is not in game
L 10/13/2019 - 16:12:01: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 16:12:01: [SM] Call stack trace:
L 10/13/2019 - 16:12:01: [SM]   [0] GetClientTeam
L 10/13/2019 - 16:12:01: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 16:12:01: [SM] Exception reported: Entity 1 (1) is invalid
L 10/13/2019 - 16:12:01: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 16:12:01: [SM] Call stack trace:
L 10/13/2019 - 16:12:01: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 16:12:01: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 16:12:01: Error log file session closed.
L 10/13/2019 - 16:12:12: SourceMod error session started
L 10/13/2019 - 16:12:12: Info (map "c2m1_highway") (file "errors_20191013.log")
L 10/13/2019 - 16:12:12: [SM] Exception reported: Client 11 is not in game
L 10/13/2019 - 16:12:12: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 16:12:12: [SM] Call stack trace:
L 10/13/2019 - 16:12:12: [SM]   [0] GetClientTeam
L 10/13/2019 - 16:12:12: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 16:15:35: [SM] Exception reported: Entity 7 (7) is invalid
L 10/13/2019 - 16:15:35: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 16:15:35: [SM] Call stack trace:
L 10/13/2019 - 16:15:35: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 16:15:35: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 16:21:46: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 16:21:46: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 16:21:46: [SM] Call stack trace:
L 10/13/2019 - 16:21:46: [SM]   [0] SetEntProp
L 10/13/2019 - 16:21:46: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 16:21:48: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 16:21:48: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 16:21:48: [SM] Call stack trace:
L 10/13/2019 - 16:21:48: [SM]   [0] SetEntProp
L 10/13/2019 - 16:21:48: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 16:22:33: [SM] Exception reported: Client 8 is not in game
L 10/13/2019 - 16:22:33: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 16:22:33: [SM] Call stack trace:
L 10/13/2019 - 16:22:33: [SM]   [0] GetClientTeam
L 10/13/2019 - 16:22:33: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 16:22:34: Error log file session closed.
L 10/13/2019 - 16:31:44: SourceMod error session started
L 10/13/2019 - 16:31:44: Info (map "c2m2_fairgrounds") (file "errors_20191013.log")
L 10/13/2019 - 16:31:44: [SM] Exception reported: Entity 9 (9) is invalid
L 10/13/2019 - 16:31:44: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 16:31:44: [SM] Call stack trace:
L 10/13/2019 - 16:31:44: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 16:31:44: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 16:45:24: Error log file session closed.
L 10/13/2019 - 16:50:30: SourceMod error session started
L 10/13/2019 - 16:50:30: Info (map "c2m3_coaster") (file "errors_20191013.log")
L 10/13/2019 - 16:50:30: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 16:50:30: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 16:50:30: [SM] Call stack trace:
L 10/13/2019 - 16:50:30: [SM]   [0] SetEntProp
L 10/13/2019 - 16:50:30: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 16:56:15: [SM] Exception reported: Entity 10 (10) is invalid
L 10/13/2019 - 16:56:15: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 16:56:15: [SM] Call stack trace:
L 10/13/2019 - 16:56:15: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 16:56:15: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 17:01:45: [SM] Exception reported: Entity 1 (1) is invalid
L 10/13/2019 - 17:01:45: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 17:01:45: [SM] Call stack trace:
L 10/13/2019 - 17:01:45: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 17:01:45: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 17:01:46: Error log file session closed.
L 10/13/2019 - 17:12:52: SourceMod error session started
L 10/13/2019 - 17:12:52: Info (map "c2m4_barns") (file "errors_20191013.log")
L 10/13/2019 - 17:12:52: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 17:12:52: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 17:12:52: [SM] Call stack trace:
L 10/13/2019 - 17:12:52: [SM]   [0] SetEntProp
L 10/13/2019 - 17:12:52: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 17:14:13: Error log file session closed.
L 10/13/2019 - 17:15:00: SourceMod error session started
L 10/13/2019 - 17:15:00: Info (map "c2m5_concert") (file "errors_20191013.log")
L 10/13/2019 - 17:15:00: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 17:15:00: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 17:15:00: [SM] Call stack trace:
L 10/13/2019 - 17:15:00: [SM]   [0] SetEntProp
L 10/13/2019 - 17:15:00: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 17:23:02: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 17:23:02: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 17:23:02: [SM] Call stack trace:
L 10/13/2019 - 17:23:02: [SM]   [0] SetEntProp
L 10/13/2019 - 17:23:02: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 17:23:31: [SM] Exception reported: Entity 4 (4) is invalid
L 10/13/2019 - 17:23:31: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 17:23:31: [SM] Call stack trace:
L 10/13/2019 - 17:23:31: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 17:23:31: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 17:23:31: [SM] Exception reported: Entity 5 (5) is invalid
L 10/13/2019 - 17:23:31: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 17:23:31: [SM] Call stack trace:
L 10/13/2019 - 17:23:31: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 17:23:31: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 17:23:31: Error log file session closed.
L 10/13/2019 - 17:23:34: SourceMod error session started
L 10/13/2019 - 17:23:34: Info (map "c1m1_hotel") (file "errors_20191013.log")
L 10/13/2019 - 17:23:34: [SM] Exception reported: Client 9 is not in game
L 10/13/2019 - 17:23:34: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 17:23:34: [SM] Call stack trace:
L 10/13/2019 - 17:23:34: [SM]   [0] GetClientTeam
L 10/13/2019 - 17:23:34: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 17:29:39: [SM] Exception reported: Client 4 is not in game
L 10/13/2019 - 17:29:39: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 17:29:39: [SM] Call stack trace:
L 10/13/2019 - 17:29:39: [SM]   [0] GetClientTeam
L 10/13/2019 - 17:29:39: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 17:30:40: Error log file session closed.
L 10/13/2019 - 17:40:00: SourceMod error session started
L 10/13/2019 - 17:40:00: Info (map "c1m2_streets") (file "errors_20191013.log")
L 10/13/2019 - 17:40:00: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 17:40:00: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 17:40:00: [SM] Call stack trace:
L 10/13/2019 - 17:40:00: [SM]   [0] SetEntProp
L 10/13/2019 - 17:40:00: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 17:48:36: [SM] Exception reported: Entity 3 (3) is invalid
L 10/13/2019 - 17:48:36: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 17:48:36: [SM] Call stack trace:
L 10/13/2019 - 17:48:36: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 17:48:36: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 17:48:36: [SM] Exception reported: Client 3 is not in game
L 10/13/2019 - 17:48:36: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 17:48:36: [SM] Call stack trace:
L 10/13/2019 - 17:48:36: [SM]   [0] GetClientTeam
L 10/13/2019 - 17:48:36: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 17:48:37: Error log file session closed.
L 10/13/2019 - 17:50:40: SourceMod error session started
L 10/13/2019 - 17:50:40: Info (map "c1m3_mall") (file "errors_20191013.log")
L 10/13/2019 - 17:50:40: [SM] Exception reported: Client index 55 is invalid
L 10/13/2019 - 17:50:40: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 17:50:40: [SM] Call stack trace:
L 10/13/2019 - 17:50:40: [SM]   [0] IsClientInGame
L 10/13/2019 - 17:50:40: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 17:50:40: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:01:38: Error log file session closed.
L 10/13/2019 - 18:03:51: SourceMod error session started
L 10/13/2019 - 18:03:51: Info (map "c1m4_atrium") (file "errors_20191013.log")
L 10/13/2019 - 18:03:51: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 18:03:51: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 18:03:51: [SM] Call stack trace:
L 10/13/2019 - 18:03:51: [SM]   [0] SetEntProp
L 10/13/2019 - 18:03:51: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 18:10:11: [SM] Exception reported: Entity 1 (1) is invalid
L 10/13/2019 - 18:10:11: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 18:10:11: [SM] Call stack trace:
L 10/13/2019 - 18:10:11: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 18:10:11: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 18:10:11: [SM] Exception reported: Entity 2 (2) is invalid
L 10/13/2019 - 18:10:11: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 18:10:11: [SM] Call stack trace:
L 10/13/2019 - 18:10:11: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 18:10:11: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 18:10:12: Error log file session closed.
L 10/13/2019 - 18:11:06: SourceMod error session started
L 10/13/2019 - 18:11:06: Info (map "c1m1_hotel") (file "errors_20191013.log")
L 10/13/2019 - 18:11:06: [SM] Exception reported: Entity 5 (5) is invalid
L 10/13/2019 - 18:11:06: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 18:11:06: [SM] Call stack trace:
L 10/13/2019 - 18:11:06: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 18:11:06: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 18:11:15: Error log file session closed.
L 10/13/2019 - 18:12:55: SourceMod error session started
L 10/13/2019 - 18:12:55: Info (map "c8m1_apartment") (file "errors_20191013.log")
L 10/13/2019 - 18:12:55: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 18:12:55: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 18:12:55: [SM] Call stack trace:
L 10/13/2019 - 18:12:55: [SM]   [0] SetEntProp
L 10/13/2019 - 18:12:55: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 18:15:05: Error log file session closed.
L 10/13/2019 - 18:18:50: SourceMod error session started
L 10/13/2019 - 18:18:50: Info (map "c8m2_subway") (file "errors_20191013.log")
L 10/13/2019 - 18:18:50: [SM] Exception reported: Client index 146 is invalid
L 10/13/2019 - 18:18:50: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 18:18:50: [SM] Call stack trace:
L 10/13/2019 - 18:18:50: [SM]   [0] IsClientInGame
L 10/13/2019 - 18:18:50: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 18:18:50: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:18:51: [SM] Exception reported: Client index 146 is invalid
L 10/13/2019 - 18:18:51: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 18:18:51: [SM] Call stack trace:
L 10/13/2019 - 18:18:51: [SM]   [0] IsClientInGame
L 10/13/2019 - 18:18:51: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 18:18:51: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:18:52: [SM] Exception reported: Client index 146 is invalid
L 10/13/2019 - 18:18:52: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 18:18:52: [SM] Call stack trace:
L 10/13/2019 - 18:18:52: [SM]   [0] IsClientInGame
L 10/13/2019 - 18:18:52: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 18:18:52: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:19:03: [SM] Exception reported: Client index 146 is invalid
L 10/13/2019 - 18:19:03: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 18:19:03: [SM] Call stack trace:
L 10/13/2019 - 18:19:03: [SM]   [0] IsClientInGame
L 10/13/2019 - 18:19:03: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 18:19:03: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:19:05: [SM] Exception reported: Client index 146 is invalid
L 10/13/2019 - 18:19:05: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 18:19:05: [SM] Call stack trace:
L 10/13/2019 - 18:19:05: [SM]   [0] IsClientInGame
L 10/13/2019 - 18:19:05: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 18:19:05: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:19:06: [SM] Exception reported: Client index 146 is invalid
L 10/13/2019 - 18:19:06: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 18:19:06: [SM] Call stack trace:
L 10/13/2019 - 18:19:06: [SM]   [0] IsClientInGame
L 10/13/2019 - 18:19:06: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 18:19:06: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:19:06: [SM] Exception reported: Client index 146 is invalid
L 10/13/2019 - 18:19:06: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 18:19:06: [SM] Call stack trace:
L 10/13/2019 - 18:19:06: [SM]   [0] IsClientInGame
L 10/13/2019 - 18:19:06: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 18:19:06: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:19:09: [SM] Exception reported: Client index 146 is invalid
L 10/13/2019 - 18:19:09: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 18:19:09: [SM] Call stack trace:
L 10/13/2019 - 18:19:09: [SM]   [0] IsClientInGame
L 10/13/2019 - 18:19:09: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 18:19:09: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:24:53: [SM] Exception reported: Client 7 is not in game
L 10/13/2019 - 18:24:53: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 18:24:53: [SM] Call stack trace:
L 10/13/2019 - 18:24:53: [SM]   [0] GetClientTeam
L 10/13/2019 - 18:24:53: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 18:24:58: Error log file session closed.
L 10/13/2019 - 18:25:19: SourceMod error session started
L 10/13/2019 - 18:25:19: Info (map "c8m3_sewers") (file "errors_20191013.log")
L 10/13/2019 - 18:25:19: [SM] Exception reported: Client 7 is not in game
L 10/13/2019 - 18:25:19: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 18:25:19: [SM] Call stack trace:
L 10/13/2019 - 18:25:19: [SM]   [0] GetClientTeam
L 10/13/2019 - 18:25:19: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 18:32:12: [SM] Exception reported: Entity 7 (7) is invalid
L 10/13/2019 - 18:32:12: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 18:32:12: [SM] Call stack trace:
L 10/13/2019 - 18:32:12: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 18:32:12: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 18:32:15: Error log file session closed.
L 10/13/2019 - 18:43:47: SourceMod error session started
L 10/13/2019 - 18:43:47: Info (map "c8m5_rooftop") (file "errors_20191013.log")
L 10/13/2019 - 18:43:47: [SM] Exception reported: Client index 222 is invalid
L 10/13/2019 - 18:43:47: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 18:43:47: [SM] Call stack trace:
L 10/13/2019 - 18:43:47: [SM]   [0] IsClientInGame
L 10/13/2019 - 18:43:47: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 18:43:47: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:43:47: [SM] Exception reported: Client index 222 is invalid
L 10/13/2019 - 18:43:47: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 18:43:47: [SM] Call stack trace:
L 10/13/2019 - 18:43:47: [SM]   [0] IsClientInGame
L 10/13/2019 - 18:43:47: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 18:43:47: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 18:48:25: Error log file session closed.
L 10/13/2019 - 18:50:17: SourceMod error session started
L 10/13/2019 - 18:50:17: Info (map "c5m1_waterfront") (file "errors_20191013.log")
L 10/13/2019 - 18:50:17: [SM] Exception reported: Entity 1 (1) is invalid
L 10/13/2019 - 18:50:17: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 18:50:17: [SM] Call stack trace:
L 10/13/2019 - 18:50:17: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 18:50:17: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 18:50:17: [SM] Exception reported: Entity 4 (4) is invalid
L 10/13/2019 - 18:50:17: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 18:50:17: [SM] Call stack trace:
L 10/13/2019 - 18:50:17: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 18:50:17: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 18:50:22: Error log file session closed.
L 10/13/2019 - 22:16:20: SourceMod error session started
L 10/13/2019 - 22:16:20: Info (map "c2m1_highway") (file "errors_20191013.log")
L 10/13/2019 - 22:16:20: [SM] Exception reported: Client 7 is not in game
L 10/13/2019 - 22:16:20: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 22:16:20: [SM] Call stack trace:
L 10/13/2019 - 22:16:20: [SM]   [0] GetClientTeam
L 10/13/2019 - 22:16:20: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 22:16:34: [SM] Exception reported: Client 7 is not in game
L 10/13/2019 - 22:16:34: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 22:16:34: [SM] Call stack trace:
L 10/13/2019 - 22:16:34: [SM]   [0] GetClientTeam
L 10/13/2019 - 22:16:34: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 22:17:05: Error log file session closed.
L 10/13/2019 - 22:19:38: SourceMod error session started
L 10/13/2019 - 22:19:38: Info (map "c5m1_waterfront") (file "errors_20191013.log")
L 10/13/2019 - 22:19:38: [SM] Exception reported: Client index 445 is invalid
L 10/13/2019 - 22:19:38: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 22:19:38: [SM] Call stack trace:
L 10/13/2019 - 22:19:38: [SM]   [0] IsClientInGame
L 10/13/2019 - 22:19:38: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 22:19:38: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 22:19:39: [SM] Exception reported: Client index 445 is invalid
L 10/13/2019 - 22:19:39: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/13/2019 - 22:19:39: [SM] Call stack trace:
L 10/13/2019 - 22:19:39: [SM]   [0] IsClientInGame
L 10/13/2019 - 22:19:39: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/13/2019 - 22:19:39: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/13/2019 - 22:22:16: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 22:22:16: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 22:22:16: [SM] Call stack trace:
L 10/13/2019 - 22:22:16: [SM]   [0] SetEntProp
L 10/13/2019 - 22:22:16: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 22:25:47: [SM] Exception reported: Entity 1 (1) is invalid
L 10/13/2019 - 22:25:47: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 22:25:47: [SM] Call stack trace:
L 10/13/2019 - 22:25:47: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 22:25:47: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 22:25:47: Error log file session closed.
L 10/13/2019 - 22:31:33: SourceMod error session started
L 10/13/2019 - 22:31:33: Info (map "c5m2_park") (file "errors_20191013.log")
L 10/13/2019 - 22:31:33: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 22:31:33: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 22:31:33: [SM] Call stack trace:
L 10/13/2019 - 22:31:33: [SM]   [0] SetEntProp
L 10/13/2019 - 22:31:33: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 22:38:54: Error log file session closed.
L 10/13/2019 - 22:59:35: SourceMod error session started
L 10/13/2019 - 22:59:35: Info (map "c5m4_quarter") (file "errors_20191013.log")
L 10/13/2019 - 22:59:35: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 22:59:35: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 22:59:35: [SM] Call stack trace:
L 10/13/2019 - 22:59:35: [SM]   [0] SetEntProp
L 10/13/2019 - 22:59:35: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 23:02:28: [SM] Exception reported: Entity 3 (3) is invalid
L 10/13/2019 - 23:02:28: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 23:02:28: [SM] Call stack trace:
L 10/13/2019 - 23:02:28: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 23:02:28: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 23:02:28: Error log file session closed.
L 10/13/2019 - 23:03:45: SourceMod error session started
L 10/13/2019 - 23:03:45: Info (map "c5m5_bridge") (file "errors_20191013.log")
L 10/13/2019 - 23:03:45: [SM] Exception reported: Client 1 is not in game
L 10/13/2019 - 23:03:45: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 23:03:45: [SM] Call stack trace:
L 10/13/2019 - 23:03:45: [SM]   [0] GetClientTeam
L 10/13/2019 - 23:03:45: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 23:07:47: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 23:07:47: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 23:07:47: [SM] Call stack trace:
L 10/13/2019 - 23:07:47: [SM]   [0] SetEntProp
L 10/13/2019 - 23:07:47: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 23:08:07: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 23:08:07: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 23:08:07: [SM] Call stack trace:
L 10/13/2019 - 23:08:07: [SM]   [0] SetEntProp
L 10/13/2019 - 23:08:07: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 23:08:59: [SM] Exception reported: Client 7 is not in game
L 10/13/2019 - 23:08:59: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 23:08:59: [SM] Call stack trace:
L 10/13/2019 - 23:08:59: [SM]   [0] GetClientTeam
L 10/13/2019 - 23:08:59: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 23:08:59: Error log file session closed.
L 10/13/2019 - 23:15:36: SourceMod error session started
L 10/13/2019 - 23:15:36: Info (map "c2m1_highway") (file "errors_20191013.log")
L 10/13/2019 - 23:15:36: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 23:15:36: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 23:15:36: [SM] Call stack trace:
L 10/13/2019 - 23:15:36: [SM]   [0] SetEntProp
L 10/13/2019 - 23:15:36: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 23:23:08: [SM] Exception reported: Entity 10 (10) is invalid
L 10/13/2019 - 23:23:08: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 23:23:08: [SM] Call stack trace:
L 10/13/2019 - 23:23:08: [SM]   [0] GetEntDataEnt2
L 10/13/2019 - 23:23:08: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/13/2019 - 23:23:09: Error log file session closed.
L 10/13/2019 - 23:23:57: SourceMod error session started
L 10/13/2019 - 23:23:57: Info (map "c2m2_fairgrounds") (file "errors_20191013.log")
L 10/13/2019 - 23:23:57: [SM] Exception reported: Client 10 is not in game
L 10/13/2019 - 23:23:57: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 23:23:57: [SM] Call stack trace:
L 10/13/2019 - 23:23:57: [SM]   [0] GetClientTeam
L 10/13/2019 - 23:23:57: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 23:38:24: [SM] Exception reported: Client 1 is not in game
L 10/13/2019 - 23:38:24: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 23:38:24: [SM] Call stack trace:
L 10/13/2019 - 23:38:24: [SM]   [0] GetClientTeam
L 10/13/2019 - 23:38:24: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 23:39:35: Error log file session closed.
L 10/13/2019 - 23:46:31: SourceMod error session started
L 10/13/2019 - 23:46:31: Info (map "c2m3_coaster") (file "errors_20191013.log")
L 10/13/2019 - 23:46:31: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 23:46:31: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 23:46:31: [SM] Call stack trace:
L 10/13/2019 - 23:46:31: [SM]   [0] SetEntProp
L 10/13/2019 - 23:46:31: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 23:46:38: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/13/2019 - 23:46:38: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/13/2019 - 23:46:38: [SM] Call stack trace:
L 10/13/2019 - 23:46:38: [SM]   [0] SetEntProp
L 10/13/2019 - 23:46:38: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/13/2019 - 23:53:24: [SM] Exception reported: Client 3 is not in game
L 10/13/2019 - 23:53:24: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 23:53:24: [SM] Call stack trace:
L 10/13/2019 - 23:53:24: [SM]   [0] GetClientTeam
L 10/13/2019 - 23:53:24: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/13/2019 - 23:56:20: [SM] Exception reported: Client 7 is not in game
L 10/13/2019 - 23:56:20: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/13/2019 - 23:56:20: [SM] Call stack trace:
L 10/13/2019 - 23:56:20: [SM]   [0] GetClientTeam
L 10/13/2019 - 23:56:20: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
